components/match2/commons/match_group_input.lua

```diff
@@ -23,6 +23,34 @@ local globalVars = PageVariableNamespace()
 
 local MatchGroupInput = {}
 
+function MatchGroupInput.readSinglematch(bracketId, args)
+	local sectionHeader = args.section or Variables.varDefault('bracket_header') or ''
+	Variables.varDefine('bracket_header', sectionHeader)
+	local tournamentParent = Variables.varDefault('tournament_parent', '')
+
+	local matches = {}
+	local matchArgs = args[1]
```

Comment:
imo we definitely should allow `args[1]` as input, but we maybe could add an alisas
suggestions:
* `args.M`
* `args.Match`
* `args.match`

Reply:
Not relevant anymore due to rework, SingleMatch is now only a display method and not a parsing/storage function.